### PR TITLE
feat(icons): added new alignment icons

### PR DESCRIPTION
**Pull request by yorbi-dp**
# feat: added new alignment icons

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [x] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1507

## :book: Description

add:
- align left
- align right
- align center
- align justify

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [ ] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

### CODE REVIEW

**Review by juliodialpad — APPROVED** [review]
Thanks!

File: packages/dialtone-icons/src/keywords.json [hunk 1]
```diff
@@ -1194,6 +1194,10 @@
       ]
     },
     "editing": {
+      "align-center": [],
+      "align-justify": [],
+      "align-left": [],
+      "align-right": [],
```
Comment (francisrupert) [inline packages/dialtone-icons/src/keywords.json]: Some keywords...

Text
Format
Flush

**Review by francisrupert — APPROVED** [review]
Very cool. Thanks for quick turnaround! And of course add them to [Figma library](https://www.figma.com/file/zz40wi0uW9MvaJ5RuhcRZR/DT9-Icon-Library?type=design&node-id=9393-1908&mode=design&t=EMcbf7E3qodDAbXl-11).

File: packages/dialtone-icons/src/keywords.json (hunk 1 above)
Comment (yorbi-dp) [reply packages/dialtone-icons/src/keywords.json]: Thanks!

**Pull request merged**